.github/workflows/london_taxi_ci_pipeline.yml

```diff
@@ -4,12 +4,13 @@ on:
     branches:
       - development
     paths:
-      - '.github/**'
-      - 'mlops/common/**'
-      - 'mlops/london_taxi/**'
-      - 'model/london_taxi/**'
-      - 'src/london_src/**'
-      - 'test/london_taxi/**'
+      - .github/*
```

Comment:
I am not sure if this pattern (with one star) will trigger any flows. Usually one * doesn't work in GitHub

Reply:
Removed  .github and added back **